pubspec.yaml

```diff
@@ -8,5 +8,5 @@ environment:
   sdk: '>=2.0.0-dev.56.0 <2.0.0'
```

Comment:
Update the upper bound to `<3.0.0` while you're at it?

Reply:
Done.